Makefile

```diff
@@ -65,14 +65,16 @@ artifact: build pack docs
 	cp -r docs/out artifacts/docs
 
 lint: $(NPM_INSTALL_STAMP) build
-	$(ESLINT) .
-	$(TSC) --noEmit --project tsconfig.json
-	$(PRETTIER) --check .
+	# $(ESLINT) .
+	# $(TSC) --noEmit --project tsconfig.json
+	# $(PRETTIER) --check .
```

Comment:
LGTM, but we should make sure we uncomment out these other lint tasks before final merge.

Reply:
Oops. Thanks for catching this, I'll fix it before merging.